src/charm.py

```diff
@@ -9,19 +9,28 @@
 import logging
 import os
 
-# Nginx Ingress Integrator
+from charms.grafana_k8s.v0.grafana_dashboard import GrafanaDashboardProvider
+from charms.loki_k8s.v0.loki_push_api import LogProxyConsumer
 from charms.nginx_ingress_integrator.v0.ingress import IngressRequires
+from charms.prometheus_k8s.v0.prometheus_scrape import MetricsEndpointProvider
 from jinja2 import Environment, FileSystemLoader
-from ops import framework, lib, main
+from ops import lib, main
 from ops.charm import CharmBase
 from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus, WaitingStatus
 
 import relations
 from log import log_event_handler
+from state import State
+
+VALID_LOG_LEVELS = ["info", "debug", "warning", "error", "critical"]
+LOG_FILE = "/var/log/temporal"
 
 logger = logging.getLogger(__name__)
 pgsql = lib.use("pgsql", 1, "postgresql-charmers@lists.launchpad.net")
 
+SERVER_PORT = 7233
```

Comment:
should this be configurable at some point?

Reply:
This will become configurable but will require some work later on across the three Temporal charms.